aiotg/bot.py

```diff
@@ -78,6 +82,42 @@ def no_handle(mt):
             )
             self._process_updates(updates)
 
+    def webhook_loop(self, loop):
+        """
+        Starts aiohttp web server.
+        """
+        app = web.Application(loop=loop)
+
+        async def handle(request):
+            update = await request.json()
+            self._process_update(update)
+            return web.Response()
+        app.router.add_route('POST', urlparse(self.webhook_url).path, handle)
+        web.run_app(app)
```

Comment:
Almost forgot! Would be nice to pass port to run_app(), I guess it can be extracted it from webhook_url.